server/mediaserver.go

```diff
@@ -49,6 +51,7 @@ var HLSWaitTime = time.Second * 45
 var BroadcastPrice = big.NewInt(1)
 var BroadcastJobVideoProfiles = []ffmpeg.VideoProfile{ffmpeg.P240p30fps4x3, ffmpeg.P360p30fps16x9}
 var MinDepositSegmentCount = int64(75) //5 mins assuming 4s segments
+var VideoStartTimes map[string]time.Time = map[string]time.Time{}
```

Comment:
I think we can avoid the bookkeeping / state management here by submitting only one timestamp field in `LogStreamEndedEvent` .

Reply:
Should have explained better.  Since the manifest IDs are not unique (for example, currently we are basically creating new streams with the same manifestID every time), the "unique manifest ID" is `{manifestID}{createdTime}`.  This means we need to keep the `createdTime` around when we send other events about the same stream.